src/test/java/edu/indiana/dlib/amppd/service/BatchServiceTests.java

```diff
@@ -264,6 +264,26 @@ public void shouldBeInvalidCollectionName() throws Exception {
         Assert.assertTrue(response.hasErrors());
 	}
 
+	/*
+	 * Deactivated collection.  Should fail validation
+	 */
+	@Test
+	public void shouldBeDeactivatedCollection() throws Exception {
+		String fileName = "batch_manifest_for_testing.csv";
+		ClassLoader classLoader = ClassLoader.getSystemClassLoader();
```

Comment:
This csv file should be in the repo, right? I wonder why it's not in the PR. Please include it if you forgot. Also, can you put it under test/resources where other test files are?